lib/AddDependencies.js

```diff
@@ -8,6 +8,7 @@ class AddDependencies {
     this.dependencies = [];
     this.target = 'dependencies';
     this.overwrite = true;
+    this.packageDir = './package.json';
```

Comment:
it's not a dir. Please, use the appropriate variable name. Maybe, `packageFilePath` would be nice.

Reply:
addressed